### ISSUE TITLE
Speed-Up 23,76 to 25fps

### ISSUE THREAD

**Issue by CaptSonic** [donmelton/other_video_transcoding#102]
Hi.

Is there any way to change the frame rate? 
In my case it's 23.976 fps to 25 fps including the audio tracks and subtitles?
Unfortunately, with many BD the pitch of the German audio tracks is not correct.

If not, could this be built in?

**Comment by donmelton** [comment]
@CaptSonic Yes, just use the `--rate` option described in `--help more`. However, it only affects the video. If the pitch of the audio is not correct, you'll have to apply some other kind of transform to the input before transcoding. Sorry about that.

**Comment by CaptSonic** [comment]
@donmelton Thanks for your quick reply. Is there any chance that --rate will affect audio and subtitles soon?

**Comment by donmelton** [comment]
@CaptSonic No. As far as I know, there's no way to do that in `ffmpeg`. Video frames rates don't apply to audio streams. It doesn't work that way. Sorry.

**Comment by CaptSonic** [comment]
Thanks for your answer.
Could ' -filter:a "atempo=1.0427094" ' be a possibility?

1.0427094 = (24000 / 1001) / 23

**Comment by donmelton** [comment]
@CaptSonic Interesting. Does that actually work for you in this case?

**Comment by donmelton** [comment]
@CaptSonic I looked into the `atempo` filter and I believe it would clearly cause audio to be out of synchronization with video for any non-trivial duration. So I don't think this is the answer.
